gamemodes/horde/gamemode/sv_economy.lua

```diff
@@ -228,6 +228,28 @@ function plymeta:Horde_DropMoney(amount)
     end
 end
 
+function plymeta:Horde_PayPlayer(plyToPay, amount)
```

Comment:
you should add a cooldown to the command, otherwise you can spam other peoples chat / be an annoyance

Reply:
how should I go about that